src/ol/source/VectorTile.js

```diff
@@ -123,10 +123,9 @@ class VectorTile extends UrlTile {
     this.overlaps_ = options.overlaps == undefined ? true : options.overlaps;
 
     /**
-       * @protected
-       * @type {function(new: Tile, import("../tilecoord.js").TileCoord, TileState, string,
-       *        import("../format/Feature.js").default, import("../Tile.js").LoadFunction)}
-       */
+     * @protected
+     * @type {import("../VectorTile.js").TileClass}
```

Comment:
Couldn't this just be `@type {typeof import("../VectorTile.js").default}`? Then the `TileClass` typedef in `ol/VectorTile` could be removed.

Reply:
Updated per your suggestion.